<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open Street Map Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/mapIcon</normaloff>:/icons/mapIcon</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="PaintingWidget" name="paintWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>487</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_AboutQt"/>
   </widget>
   <widget class="QMenu" name="menuOSM">
    <property name="title">
     <string>OSM</string>
    </property>
    <addaction name="action_OSM_Directory"/>
    <addaction name="action_EnableDownloading"/>
    <addaction name="action_DownloaderInfo"/>
    <addaction name="action_DownloaderSetup"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuOSM"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_AboutQt">
   <property name="text">
    <string>&amp;About Qt</string>
   </property>
  </action>
  <action name="action_OSM_Directory">
   <property name="text">
    <string>OSM Directory</string>
   </property>
  </action>
  <action name="action_EnableDownloading">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Downloading</string>
   </property>
  </action>
  <action name="action_DownloaderInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Downloader Info</string>
   </property>
  </action>
  <action name="action_DownloaderSetup">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Downloader Setup</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PaintingWidget</class>
   <extends>QWidget</extends>
   <header>Layers/PaintingWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
